components/ui/TypedWord.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="typed-text__container" :class="{ 'typed-text__cursor': showCursor }">
+    <span class="typed-text__content">{{typeValue}}</span>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
In qiskit.org we follow TypeScript class-based components as shown in [Nuxt.js documentation](https://typescript.nuxtjs.org/cookbook/components/#template) (see "Class API" tab). Could you rewrite this component to follow this style, please?